tests/unit/handlers/deleteGuildRoleHandler.test.ts

```diff
@@ -0,0 +1,128 @@
+    const mockRequest = generateDummyRequestObject({
+      url: "/roles",
+      params: {
+        roleId: roleId,
+      },
+      query: {
+        dev: "false",
+      },
+      method: "DELETE",
+    });
+    const response = await deleteGuildRoleHandler(mockRequest, guildEnv);
+    const jsonResponse = await response.json();
+    expect(jsonResponse).toEqual(responseConstants.NOT_FOUND);
+  });
+  it("should return BAD_REQUEST when roleId is not valid", async () => {
+    const mockRequest = generateDummyRequestObject({
+      url: "/roles",
+      params: {
+        roleId: "",
+      },
+      query: {
+        dev: "true",
+      },
+      method: "DELETE",
+      headers: { Authorization: "Bearer testtoken" },
+    });
+    const response = await deleteGuildRoleHandler(mockRequest, guildEnv);
+    const jsonResponse = await response.json();
+    expect(jsonResponse).toEqual(responseConstants.BAD_REQUEST);
+  });
+  it("should return BAD_SIGNATURE when authorization header is not provided", async () => {
+    const mockRequest = generateDummyRequestObject({
+      url: "/roles",
+      params: {
+        roleId: roleId,
+      },
+      query: {
+        dev: "true",
+      },
+      method: "DELETE",
+    });
+    const response = await deleteGuildRoleHandler(mockRequest, guildEnv);
+    const jsonResponse = await response.json();
+    expect(jsonResponse).toEqual(responseConstants.BAD_SIGNATURE);
+  });
+  it("should return INTERNAL_SERVER_ERROR when response is not ok", async () => {
+    const mockResponse = responseConstants.INTERNAL_SERVER_ERROR;
+    const mockRequest = generateDummyRequestObject({
+      url: "/roles",
+      params: {
+        roleId: roleId,
+      },
+      query: {
+        dev: "true",
+      },
+      method: "DELETE",
+      headers: { Authorization: "Bearer testtoken" },
+    });
+    jest
+      .spyOn(deleteGuildRoleUtils, "deleteGuildRole")
+      .mockResolvedValue(mockResponse);
+    const response = await deleteGuildRoleHandler(mockRequest, guildEnv);
+    const jsonResponse = await response.json();
+    expect(jsonResponse).toEqual(mockResponse);
+  });
+  it("should return INTERNAL_SERVER_ERROR when token is not verified", async () => {
+    const mockResponse = responseConstants.INTERNAL_SERVER_ERROR;
+    const mockRequest = generateDummyRequestObject({
+      url: "/roles",
+      method: "DELETE",
+      params: {
+        roleId: roleId,
+      },
+      query: {
+        dev: "true",
+      },
+      headers: { Authorization: "Bearer testtoken" },
+    });
+    jest
+      .spyOn(verifyTokenUtils, "verifyNodejsBackendAuthToken")
+      .mockRejectedValue(mockResponse);
+    const response = await deleteGuildRoleHandler(mockRequest, guildEnv);
+    const jsonResponse = await response.json();
+    expect(jsonResponse).toEqual(mockResponse);
+  });
+  it("should return ok response", async () => {
+    const mockResponse = new Response(null, {
```

Comment:
Can we rename it to `expectedResponse`? It would be more appropriate

Reply:
fixed it